addons/mail/static/src/core/web/activity.js

```diff
@@ -72,6 +72,10 @@ export class Activity extends Component {
         return computeDelay(this.props.data.date_deadline);
     }
 
+    get onUpdate() {
+        return this.props.onUpdate?.();
```

Comment:
I don't think it's correct? Calling `this.onUpdate()` like done before will crash (at least if `props.onUpdate` is not passed since the default prop is a function that returns nothing. Shouldn't it be:

```js
get onUpdate() {
    return this.props.onUpdate;
}
```